src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/NuGet.Build.Tasks.Pack.Library.csproj

```diff
@@ -48,6 +51,25 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <Target Name="PublishAndMergePackNupkg" AfterTargets="AfterBuild" Condition="'$(TargetFramework)' != ''">
+    <MSBuild
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="Publish;ILRepackNuGetPack"/>
+  </Target>
+
+  <Target Name="ILRepackNuGetPack"  Condition="'$(TargetFramework)' != ''">
+    <Message Text="ILRepack: $(ILRepackExePath)" Importance="High" />
+    <ItemGroup>
+      <BuildArtifacts Include="$(OutputPath)*.dll" Exclude="$(OutputPath)*Pack.dll" />
+    </ItemGroup>
+    <PropertyGroup>
+      <PathToBuiltNuGetPack>$(OutputPath)NuGet.Build.Tasks.Pack.dll</PathToBuiltNuGetPack>
+      <IlmergeCommand>$(ILRepackExePath) $(PathToBuiltNuGetPack) /lib:$(OutputPath)$(RuntimeIdentifier)\publish @(BuildArtifacts, ' ') /out:$(OutputPath)NuGet.Build.Tasks.Pack.dll /internalize</IlmergeCommand>
```

Comment:
we haven't been shipping symbols for this, as we don't create a symbols package.